fossunited/fossunited/doctype/foss_global_cfp_review_settings/foss_global_cfp_review_settings.py

```diff
@@ -2,6 +2,7 @@
 # For license information, please see license.txt
 
 import frappe
+from document.doctype_ids import FOSS_USER_PROFILE
```

Comment:
It should be `fossunited.doctype_ids` here ? Why is document referred ?

Reply:
Please check if this fixed, the CI tests have passed yet.